apps/resources/sbt.json

```diff
@@ -3,13 +3,8 @@
     "central"
   ],
   "dependencies": [
-    "org.scala-sbt:sbt-launch:latest.stable",
-    "io.get-coursier.sbt:sbt-runner:latest.stable"
```

Comment:
I think we should keep those. These can be helpful when running `cs launch sbt`, which doesn't use prebuilt binaries for now.